server/log_test.go

```diff
@@ -147,8 +147,12 @@ func TestLogOutput(t *testing.T) {
 	d := &dummyLogger{}
 
 	checkLogger := func(output string) {
-		if d.msg != output {
-			t.Fatalf("Unexpected logger message: %v", d.msg)
+		checkMsg := output
```

Comment:
Please revert those changes. You don't need to do anything special here. I am using a dummy logger (see code above) and this is not using the normal logger. So whatever text is passed to `Noticef`, `Debugf`, etc.. in this test is set in `dummyLogger.msg`.